go.mod

```diff
@@ -49,13 +51,13 @@ require (
 	github.com/testcontainers/testcontainers-go v0.30.0
 	go.uber.org/zap v1.27.0
 	google.golang.org/api v0.172.0
-	k8s.io/api v0.29.4
-	k8s.io/apiextensions-apiserver v0.29.3
-	k8s.io/apimachinery v0.29.4
-	k8s.io/client-go v0.29.4
-	k8s.io/component-base v0.29.4
+	k8s.io/api v0.30.0
+	k8s.io/apiextensions-apiserver v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/client-go v0.30.0
+	k8s.io/component-base v0.30.0
```

Comment:
Can we please have all of those k8s related version bumps in a separate PR? 

This makes it easier to track changes, backport changes to other releases (if that's ever necessary) and track failing tests to their respective changes.

Reply:
@pmalek  Sure.  Please see https://github.com/Kong/kubernetes-ingress-controller/pull/5935